src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -63,5 +75,75 @@ public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContex
         }
 
         public override bool GetPropertiesSupported(ITypeDescriptorContext context) => true;
+
+        private unsafe Stream GetBitmapStream(byte[] rawData)
+        {
+            Debug.Assert(rawData != null, "rawData is null.");
```

Comment:
nit, this assert could go just above line 90 because that's where it will always go if rawData is null.

Reply:
sry @danmosemsft I didn't understand why it will proceed to line 90 when rawData is null. Can you please explain ?